modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
+* In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab, and then click *Invite users*.
+* Navigate to your profile icon, and then click *Invite users*.
+
+. In the *Invite users* dialog box, provide the following information:
+* *Emails to invite*: Enter one or more email addresses of the users you want to invite. Ensure that they are valid email addresses associated with the intended recipients.
+* *Provider*: From the drop-down list, select a provider you want to use for each invited user.
++
+[IMPORTANT]
+====
+* If you have only one authentication provider available, it is selected by default.
+* If multiple authentication providers are available and at least one of them is `Red Hat SSO` or `Default Internal SSO`, that provider is selected by default.
+* If multiple authentication providers are available, but none of them is `Red Hat SSO` or `Default Internal SSO`, you are prompted to select one manually.
+* If you have not yet set up authentication providers, you will receive a warning message; the form will be disabled and you will receive a link that takes you to the *Access Control* section where you can configure an authentication provider.
+====
+* *Role*: From the drop-down list, select the role to assign to each invited user.
+
+. Click *Invite users*.
+. When you receive confirmation that the users have been created with the selected role, copy the one or more email addresses and the message into an email that you create in your own email client, and send it to the users.
+. Click *Done*.
+
+.Verification
+
+. In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab.
+. Select the authentication provider you used to invite one or more users.
```

Comment:
Even though it could be "one or more" I don't think you need to specify that anymore. I think you can just generically say "users" now. Here in this step and in step 4 of verification.

```suggestion
. Select the authentication provider you used to invite users.
```